app/packages/relay/src/queries/viewSchema.ts

```diff
@@ -0,0 +1,16 @@
+import { graphql } from "react-relay";
+
+import r from "../resolve";
+
+export default r(graphql`
+  fragment viewSchemaFragment on Query
+  @refetchable(queryName: "viewSchemaFragmentQuery") {
+    schemaForViewStages(datasetName: $name, viewStages: $viewStages) {
```

Comment:
Not import right now, but I think this could be consolidated into the Dataset type

Reply:
I was hoping to include it as view_schema in the dataset type but unfortunately the schema was not the one I expected. we should talk offline about this actually